StreamChatUITestsAppUITests/MockServer/StreamMockServer.swift

```diff
@@ -11,13 +11,11 @@ final class StreamMockServer {
     // Delays all HTTP responses by given time interval, 0 by default
     static var httpResponseDelay: TimeInterval = 0.0
 
-    // This constant is used inside `websocketDelay` func that delays websocket requests and responses by given time interval, 1.5 by default
-    var websocketDelay: TimeInterval = 1.5
-
     private(set) var server: HttpServer = HttpServer()
     private weak var globalSession: WebSocketSession?
     private var channelConfigs = ChannelConfigs()
     var messageList: [[String: Any]] = []
+    var ephemeralMessageList: [[String: Any]] = []
```

Comment:
@alteral why do we store ephemeral messages separately?

Reply:
Good question. `ephemeralMessageList` is a list of ephemeral messages that were not sent yet. Once they are sent, they will be merged into the original `messageList` and transformed into `regular` messages.

Reply:
The fact we store them separately requires 2 pairs of `save/find/remove` methods, one for regular and one for ephemeral messages. So I'm wondering what was the limitation of storing ephemeral messages in the same array?

Reply:
Updated, no more `ephemeralMessageList`, thanks a mil! 🙂